test/test_seqno.cpp

```diff
@@ -41,6 +41,12 @@ TEST(CSeqNo, seqoff)
     EXPECT_EQ(CSeqNo::seqoff(0x7FFFFFFF, 1),    2);
 }
 
+TEST(CSeqNo, seqlen)
+{
+    EXPECT_EQ(CSeqNo::seqlen(125, 125), 1);
+    EXPECT_EQ(CSeqNo::seqlen(125, 126), 2);
```

Comment:
I have a better idea for a test:

```
const int startseq = 10;
const int npackets = 5;

int sndlastack = startseq + 1; // initialized as one past the last known sequence
int sndcurrseqno = startseq + npackets;

EXPECT_EQ(CUDT::flightspan(sndlastack, sndcurrseqno), npackets);
sndlastack += npackets;
EXPECT_EQ(CUDT::flightspan(sndlastack, sndcurrseqno), 0);
```

Reply:
Added `CUDT.getFlightSpan` unit test.

Reply:
I have written this kind of test not without the reason.

This resembles the procedure of increasing the span when sending packets and wiping it out when acknowledging.

Reply:
Of course. However, the test does not need to reproduce the logic of sequence increment. It only needs to check the function works correctly. Listing target values to check is more readable.

```c++
TEST(CUDT, getFlightSpan)
{
    const int test_values[3][3] = {
        // lastack  curseq  span
        {      125,    124,   0 },
        {      125,    125,   1 },
        {      125,    130,   6 }
    };

    for (auto val : test_values)
    {
        EXPECT_EQ(CUDT::getFlightSpan(val[0], val[1]), val[2]) << "Span(" << val[0] << ", " << val[1] << ")";
    }
}
```